nestjs/src/courses/mentors/dto/mentor-options.dto.ts

```diff
@@ -0,0 +1,50 @@
+import { Student } from '@entities/index';
+import { Mentor } from '@entities/mentor';
+import { PreferredStudentsLocation } from '@common/enums/mentor';
+import { ApiProperty, ApiResponse } from '@nestjs/swagger';
+import { IsArray, IsNumber, IsString } from 'class-validator';
+import { PersonDto } from 'src/core/dto';
+
+class StudentsDto {
+  @ApiProperty()
+  @IsNumber()
+  id: number;
+
+  @ApiProperty()
+  @IsString()
+  githubId: string;
+
+  @ApiProperty()
+  @IsString()
+  name: string;
+
+  constructor(student: Student & { user: { githubId: string; firstName: string; lastName: string } }) {
```

Comment:
Why don't you extract that student type, since you're using it few times down bellow?

Reply:
thx, done